### PR TITLE
Iceworks/release changelog

### PR DESCRIPTION
**Pull request by chenbin92**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+## 3.0.0-beta.8
+
+- [fix] redirect to task page when run task #2495
+- [fix] fix vue materials don't appear in material panel #2496
+- [fix] feat: add material page loading #2500
```
Comment (alvinhui) [inline CHANGELOG.md]: [feat] add material page loading

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+- [fix] use ice-npm-utils in iceworks-cli #2506
```
Comment (alvinhui) [inline CHANGELOG.md]: [refactor]

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+- [fix] format dependency log #2499
+- [fix] log info in production environment #2490
```
Comment (alvinhui) [inline CHANGELOG.md]: feat

File: CHANGELOG.md (hunk 1 above)
Comment (chenbin92) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (chenbin92) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 3 above)
Comment (chenbin92) [reply CHANGELOG.md]: fixed

**Pull request merged**